frontend/smart-kickers-game/src/App.js

```diff
@@ -32,12 +36,24 @@ function App() {
   const handleEndGame = () => {
     setFinalScores({ blue: blueScore, white: whiteScore });
     setIsStatisticsDisplayed(!isStatisticsDisplayed);
+    getHeatmap();
   };
+  async function getHeatmap() {
+    //Todo uncomment this line when backend is working
+    let heatMapTable = await getHeatmapData();
+    setHeatmap(heatMapTable.data.Heatmap);
+  }
+
```

Comment:
Firstly, I would recommend to move the whole logic related to the Heatmap out of the `App` component.

Reply:
Where should this function be moved?

Reply:
Done. Heatmap logic has been moved to custom useHeatmap hook.